sibylapp/config.py

```diff
@@ -13,6 +13,7 @@
 MAX_ENTITIES = cfg.get("MAX_ENTITIES", 11)
 DATASET_SIZE = cfg.get("DATASET_SIZE", 1000)
 LOAD_UPFRONT = cfg.get("LOAD_UPFRONT", True)
+USE_ROWS = cfg.get("USE_ROWS", True)
```

Comment:
Since we need to test on housing and brakepad data for every PR, our quality of life is much improved by reducing the number of configs we need to change between these two datsets.
I think we should set this to have an automatic functionality when this config is not give. ie, the row selector only appears if the selected entity has more than one row